src/routes/articles.js

```diff
@@ -17,15 +18,18 @@ const {
 	 deletePostSchema,
 	 assignTagToArticleSchema,
 	 deleteArticleTagsSchema,
-	 queryArticleTagsSchema
+	 queryArticleTagsSchema,
+	 recordLikesOnPostSchema
```

Comment:
forgot to review this part, let's use `likePostSchema`

Reply:
yes please